Fix mobile apps intro paragraph (#27033)
DataDog/documentation: content/en/mobile/_index.md

  text: "Getting started with the Datadog mobile app"
---

> The Datadog Mobile app enables you to view alerts from Datadog on your mobile device. When receiving an alert via On-Call, Slack or e-mail, you'll be able to investigate issues by opening monitor graphs and dashboards on your mobile device.

Reviewer: ```suggestion
The Datadog Mobile app enables you to view alerts from Datadog on your mobile device. When receiving an alert through On-Call, Slack, or email, you can investigate issues by opening monitor graphs and dashboards on your mobile device.
```
